thalerj/openbmctool.py

```diff
@@ -0,0 +1,1490 @@
+#!/usr/bin/python3
+
+'''
+#================================================================================
+#
+#    openbmctool.py
+#
+#    Copyright IBM Corporation 2015-2017. All Rights Reserved
+#
+#    This program is licensed under the terms of the Eclipse Public License
+#    v1.0 as published by the Eclipse Foundation and available at
+#    http://www.eclipse.org/legal/epl-v10.html
+#
+#    U.S. Government Users Restricted Rights:  Use, duplication or disclosure
+#    restricted by GSA ADP Schedule Contract with IBM Corp.
+#
+#================================================================================
+'''
+import argparse
+import requests
+import getpass
+import json
+import os
+import urllib3
+import time, datetime
+# import yaml
+import binascii
+import subprocess
+import platform
+import zipfile
+
+#isTTY = sys.stdout.isatty()
+
+"""
+     Used to add highlights to various text for displaying in a terminal
+       
+     @param textToColor: string, the text to be colored
+     @param color: string, used to color the text red or green
+     @param bold: boolean, used to bold the textToColor
+     @return: Buffered reader containing the modified string. 
+"""    
+def hilight(textToColor, color, bold):
+    if(sys.platform.__contains__("win")):
```

Comment:
Please run `pep8` over the script: https://pypi.python.org/pypi/pep8

Python does not require parens around conditions, which is what has caught my eye.